src/endpoints/asset-endpoints.js

```diff
@@ -0,0 +1,93 @@
+const HTTPClient = require('../http-client')
+
+const bulkHeaders = {
+  'Content-Type': 'application/vnd.api+json; ext=bulk',
+  'Accept': 'application/vnd.api+json; ext=bulk'
+}
+
+/**
+ * Bulk add assets
+ *
+ * @param [{ name, reference, remote_file_url, asset_folder: { name, reference } }] assets
+ */
+function createAssetsFilesV1 (assets) {
+  const data = assets.map(asset => {
+    return {
+      type: 'asset_files',
+      attributes: {
+        name: asset.name,
+        reference: asset.reference,
+        file_remote_url: asset.remote_file_url
+      },
+      relationships: {
+        asset_folder: {
+          type: 'asset_folders',
+          attributes: {
+            name: asset.folder ? asset.folder.name : 'Product Imagery',
```

Comment:
Personally, I'd remove the default and make it a requirement to pass it and be more explicit.